src/Symfony/Component/Serializer/Tests/Fixtures/NaturalOrderPostNormalizer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Fixtures;
+
+use Symfony\Component\Serializer\Normalizer\PostNormalizerInterface;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class NaturalOrderPostNormalizer implements PostNormalizerInterface
```

Comment:
Thank you. The post normalizer example now use the `strnatcasecmp` comparison function.